docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -139,15 +191,17 @@ Starlight dapat secara otomatis membuat grup di sidebar Anda berdasarkan direkto
 Ini berguna ketika Anda tidak ingin memasukkan setiap item sidebar secara manual ke dalam grup.
 Halaman akan diurutkan secara alfabetis berdasarkan nama file secara default.
 
+Secara default, halaman diurutkan berdasarkan abjad menurut file [`slug`](/id/reference/overrides/#slug).
+
 Tambahkan grup yang dihasilkan secara otomatis menggunakan objek dengan properti `label` dan `autogenerate`. Konfigurasi `autogenerate` Anda harus menentukan `directory` yang akan digunakan untuk entri sidebar. Sebagai contoh, dengan konfigurasi berikut:
 
-```js
+```js "label:" "autogenerate:"
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Hasilkan grup tautan secara otomatis untuk direktori 'guides'.
-			autogenerate: { directory: 'guides' },
+			label: 'Constellations',
```

Comment:
This label can also be translated if it makes sense (and same goes for the example below).

Reply:
Ok, will update soon.